_data/simple-icons.json

```diff
@@ -4411,6 +4411,12 @@
             "source": "https://firebase.google.com/brand-guidelines/",
             "guidelines": "https://firebase.google.com/brand-guidelines/"
         },
+        {
+            "title": "Firefish",
+            "hex": "F6C177",
```

Comment:
```suggestion
            "hex": "F19885",
```
Using a tool like [this](https://meyerweb.com/eric/tools/color-blend/#:::hex) you can find the _rough_ midpoint of a gradient. The suggestion above is halfway between the 'gold' and 'love' colours used in the logo.
![image](https://github.com/simple-icons/simple-icons/assets/1830867/3292e209-5272-4441-8695-e3a6da2f6ab4)